site/content/en/docs/Getting Started/create-fleetautoscaler.md

```diff
@@ -30,9 +30,17 @@ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/{{
 
 You should see a successful output similar to this :
 
+{{% feature expiryVersion="0.10.0" %}}
```

Comment:
I think it should expiry at the same moment as new one published
```suggestion
{{% feature expiryVersion="0.11.0" %}}
```

Reply:
Yeees. Good point.